universal-application-tool-0.0.1/test/services/applicant/ApplicantServiceImplTest.java

```diff
@@ -29,28 +25,22 @@
 import services.program.ProgramQuestionDefinition;
 import services.program.ProgramService;
 import services.program.ProgramServiceImpl;
-import services.question.QuestionService;
-import services.question.types.CheckboxQuestionDefinition;
-import services.question.types.NameQuestionDefinition;
 import services.question.types.QuestionDefinition;
+import support.TestQuestionBank;
 
 public class ApplicantServiceImplTest extends WithPostgresContainer {
 
   private ApplicantServiceImpl subject;
   private ProgramService programService;
-  private QuestionService questionService;
-  private QuestionDefinition questionDefinition;
   private ProgramDefinition programDefinition;
   private ApplicantRepository applicantRepository;
 
   @Before
   public void setUp() throws Exception {
     subject = instanceOf(ApplicantServiceImpl.class);
     programService = instanceOf(ProgramServiceImpl.class);
-    questionService = instanceOf(QuestionService.class);
     applicantRepository = instanceOf(ApplicantRepository.class);
-    createQuestions();
-    createProgram();
+    createProgram(TestQuestionBank.applicantName().getQuestionDefinition());
```

Comment:
(blocking): next to the `TestQuestionBank` is a `ProgramBuilder` for fluently building `Program`s in tests. That should be used here.

They may be two separate issues in our backlog but the refactoring is really done best with both `TestQuestionBank` and `ProgramBuilder`.

Reply:
I'll update the issue to make this clear.